providers/openlineage/src/airflow/providers/openlineage/utils/utils.py

```diff
@@ -864,7 +877,7 @@ class AirflowContextDeprecationWarning(UserWarning):
                     # Those are deprecated values in _DEPRECATION_REPLACEMENTS
                     # in airflow.utils.context.Context
                     return "<<non-redactable: Proxy>>"
-                if name and should_hide_value_for_key(name):
+                if name and self._should_hide_value_for_key(name):
```

Comment:
cc @mobuchowski could you just check this part to see if it makes sense? Tests pass but just double checking